package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "corona-warn-app-landingpage",
   "version": "1.1.0",
```

Comment:
I have not seen this before.

Running Node.js 18 LTS generates lockfileVersion 2, not 3.

Reply:
Here is some documentation

https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json#lockfileversion

I suspect some command updated it to 3, I'm not entirely sure when it happened. I will reinvestigate.. or just change it back to 2, as that seems to be the default. Possibly it was `ncu` or the `npm outdated` and `npm update` command.